inc/thorvg.h

```diff
@@ -1339,7 +1339,17 @@ class TVG_EXPORT Initializer final
 /**
  * @class Saver
  *
- * @brief A class enabling saving a paint in a binary format.
+ * @brief A class exporting a paint in a specified file data to recover the paint scene later.
+ *
+ * ThorVG provides a feature for exporting & importing paint data, the Saver has a role to export it to a file.
```

Comment:
ThorVG provides a feature for exporting & importing paint data. The purpose of the Saver is to export the data to a file.

Reply:
updated